### DISCUSSION TITLE
OSM with vsgpagedlod

### DISCUSSION THREAD

**Discussion by bclay1297** [vsg-dev/VulkanSceneGraph#579] [General]
I finally got a chance to get back to the curl issue in vsgXchange / vsgpagedlod. 

I believe it was Francois that suggested this change to curl.cpp

    curl_easy_setopt(_curl, CURLOPT_SSL_OPTIONS, (long)CURLSSLOPT_NATIVE_CA);

The change does work and I am able to see OSM tiles in vsgpagedlod.  I did submit an issue for this fix.  I am not sure if the fix will work across all platforms of if it should be conditionally compiled for WIN32

There is still a difference between --osm and --rm in navigation
--rm appears to rotate about the center of the earth but --osm appears to rotate about thew surface of the earth,  i.e you cannot rotate the globe to the back side and keep the same distance.  I do not see anything in vsgpagedlod that sets a reference.  How can we get both tile sets to work the same.

Bruce

**Comment by robertosfield** [comment]
Good to hear there is some path forward.  Tomorrow I'll do a review of
vsgpagedlod and cirl to see about resolving the problem.
